bplan/templates/base.html

```diff
@@ -26,20 +26,20 @@
         {% verbatim %}
         <div>
             <div class="container-fluid" ng-app="app">
-                <div class="row" ng-controller="ViewController">
+                <div ng-controller="ViewController">
```

Comment:
without the row the nav moves quite a bit to the right. maybe we can add a negative marging to the nav so it stays the same?

Reply:
row here causes inconsistencies in padding and margin for the children so have just wrapped the nav in a row div